cola/__main__.py

```diff
@@ -8,6 +8,8 @@
 """
 from __future__ import absolute_import, division, unicode_literals
 
```

Comment:
Drop this blank line `__future__` is part of the regular stdlib imports so should be grouped alongside `sys`